config/config.exs

```diff
@@ -119,7 +119,8 @@ config :skate, Skate.Repo,
 config :skate, Skate.WarmUp,
   minimum_percent_queries_to_succeed: 0.6,
   max_attempts: 20,
-  seconds_between_attempts: 1
+  seconds_between_attempts: 5,
```

Comment:
note(non-blocking): Also just wanting to acknowledge that because we don't have a backoff time that scales with the number of attempts, and that WarmUp prevents the rest of the application from running until it's done (due to init/1), the application won't really start until some multiple of 5.

Maybe we consider starting the `Schedule.Fetcher` before `Repo` and `WarmUp`, so that it's got as much time as possible to complete? I don't see any references to `Skate.Repo` in `Schedule.Fetcher` so I think it'd be okay, but then that breaks up the `start_data_processes` flag.

Reply:
Really good point! And yes, you're correct that the schedule process tree shouldn't use the database at all. I think I'll go ahead and make that change.